src/app/ipv/middleware.ts

```diff
@@ -146,36 +145,12 @@ export const handleBackendResponse = async (
     }
   }
 
-  const message = {
-    description: "Unexpected backend response",
-    data: backendResponse,
```

Comment:
Are we losing the response log here?

Reply:
The Axios interceptor logs all responses, so we'll see it there instead